src/platform/platform-device-info.h

```diff
@@ -35,10 +35,10 @@ class platform_device_info : public device_info
 
     std::string get_address() const override
     {
-        for( auto & d : _group.uvc_devices )
-            return d.device_path;
-        for( auto & d : _group.usb_devices )
-            return d.id;
+        if( _group.uvc_devices.size() )
```

Comment:
Style comment (you don't have to change if you prefer this style)
I think it is clearer to use `if( ! _group.uvc_devices.empty() )`
or at least `if( _group.uvc_devices.size() > 0 )`

Reply:
I also think that the approach is more elegant and readable.